doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -974,14 +974,12 @@ on success. It is stronger than the tactic ``fold``.
 
 .. example:: :n:`innermost` and :n:`outermost`
 
-   The type of `Nat.add_comm` is `forall n m : nat, n + m = m + n`.
-
    .. coqtop:: all
 
-      Require Import Stdlib.Arith.Arith.
+      Axiom add_comm : forall n m : nat, n + m = m + n.
```

Comment:
Using `Axiom` instead of a readily proven theorem is a bad example for users.  We should not do it this way.